### PR TITLE
Add image urls to subdivisions

### PR DESCRIPTION
**Pull request by BlackYps**

### CODE REVIEW

**Review by Sheikah45 — APPROVED** [review]

**Comment by BlackYps** [comment]
Can you merge this, so I can check if the image loading works now?

**Comment by BlackYps** [comment]
I remembered that I can just change it in build.gradle for testing

**Comment by BlackYps** [comment]
Works now as expected

File: faf-commons-api/src/main/java/com/faforever/commons/api/dto/LeagueSeasonDivisionSubdivision.java [hunk 1]
```diff
@@ -6,18 +6,23 @@
...
   private String descriptionKey;
   private Integer highestScore;
   private Integer maxRating;
   private Integer minRating;
-  @ToString.Include
-  private String nameKey;
   private Integer subdivisionIndex;
+  private URL imageUrl;
+  private URL mediumImageUrl;
+  private URL smallImageUrl;
```
Comment (Brutus5000) [inline faf-commons-api/src/main/java/com/faforever/commons/api/dto/LeagueSeasonDivisionSubdivision.java]: Please use String. URL is not suitable for data transfer objects with it's retarted exception handling.

**Review by Brutus5000 — CHANGES_REQUESTED** [review]

**Comment by BlackYps** [comment]
I'll update the PR when I have found out how to handle the mapping from string to url in the client

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**